src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,24 @@
 import React from 'react';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  movies: Movie[]
+};
+
+export const MoviesList: React.FC<Props> = ({ movies = [] }) => (
```

Comment:
We need this default value?

Usually, your component will receive already empty array, so if this not required by your functionality - remove this.